### PR TITLE
An example of using the experimental WebSockets module

### PR DESCRIPTION
**Pull request by olegbespalov**
# What?

Adding the example of using the new experimental WebSockets API and the links to the GH and community forum for providing feedback/reports.

# Why?

Having an example is necessary for the documentation.
Links should help users.

### CODE REVIEW

File: src/components/shared/experimental-blockquote/experimental-blockquote.view.js [hunk 1]
```diff
@@ -7,8 +7,9 @@ const ExperimentalBlockquote = () => (
     need to add features or introduce breaking changes. This could happen at any
     time until we release this module as stable.
     <p>
-      Feel free to provide user feedback, and open an issue or pull request if
-      you have any suggestions.
+      Feel free to [provide user feedback](https://community.k6.io/), and [open
+      an issue](https://github.com/grafana/k6/issues) or pull request if you
+      have any suggestions.
```
Comment (mstoykov) [inline src/components/shared/experimental-blockquote/experimental-blockquote.view.js:12]: I don't know if it won't be better if this is linking to the xk6-websockets repo for now :thinking:

**Review by mstoykov — APPROVED** [review]

File: src/components/shared/experimental-blockquote/experimental-blockquote.view.js (hunk 1 above)
Comment (olegbespalov) [reply src/components/shared/experimental-blockquote/experimental-blockquote.view.js:12]: I don't have a strong opinion :shrug: as long as we have an established channel of getting feedback :relaxed: 

But here are my thoughts. In theory, we can link to the module's GitHub, but on the other side, we offer it as part of the k6, so issues/feature requests to the main k6 repo look fine. Also, this block is a general block for all experimental extensions.

**Comment by ppcano** [comment]
> An example below showcases how a single VU can run multiple WebSockets connections

👍 

I suggest adding also an example that aims to stress the Websocket server by sending a continuous high number of messages during the test duration.

**Review by ppcano — APPROVED** [review]

File: src/data/markdown/docs/02 javascript api/07 k6-experimental/03 websockets.md [hunk 2]
```diff
@@ -15,4 +15,67 @@ With some caveats:
 
 | Class                                       | Description                                                                                    |
 | :------------------------------------------ | :--------------------------------------------------------------------------------------------- |
-| [WebSocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/WebSocket)     | Constructs a new WebSocket connection |
\ No newline at end of file
+| [WebSocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/WebSocket)     | Constructs a new WebSocket connection |
+
+## Example
+
+An example below showcases how a single VU can run multiple WebSockets connections asynchronously and how to stop them after a period using the timeout and interval functions.
```
[the remark this answers is not in the document]
Comment (MattDodsonEnglish) [reply src/data/markdown/docs/02 javascript api/07 k6-experimental/03 websockets.md:22]: I made a slight assumption that `them` in the original refers to the connections. Please confirm.

**Review by MattDodsonEnglish — CHANGES_REQUESTED** [review]
Nice! :+1: 

Besides my tiny nit to improve scannability, it might make sense to add comments before the major functions in the `startWSworker` logic.

Even if the code is very obvious for experienced websockets programmers, this example might have a secondary audience of people who are "websockets curious". Maybe they just want to check the example out to see how complicated it looks.

Here's how I understood the code:

For the duration of the test, have a single VU that runs the following logic four times:

1. Create a chat connection and give the user a name based on the `__VU` id.
2. Listen for messages and display the content (or error)
3. Say something every 2-8 seconds (I stole this coment :slightly_smiling_face: )
4. Couldn't exactly understand how the timeouts are triggered―I'd have to read about `clearInterval` :slightly_frowning_face: 
5. Close this connection 

I'm just typing this up to provide the context of how it looks from a total outsider/amateur. Maybe comments would save time for people who just want to get the gist of how it works.

**Review by MattDodsonEnglish — APPROVED** [review]
LGTM now! :muscle:

**Pull request merged**